### PR TITLE
Configuareble external URL, fix collection permission on setup

### PR DESCRIPTION
**Pull request by MauPalantir**

### CODE REVIEW

File: post-install.xql [hunk 1]
```diff
@@ -71,13 +73,25 @@ declare function local:create-user() as empty-sequence() {
...
             string(environment-variable($opt))
-            else ()
+        else "mermeid"
 
     return if ($password) then 
         sm:create-account('mermeid', $password, 'mermeid', ('mermedit'))
     else ()
 };
 
+
```
Comment (peterstadler) [inline post-install.xql:83]: is this function really necessary? I thought the variable "MERMEID_app_url" would be handled by the generic function `local:set-options()` (in that same file) and thus updating the properties file?

Comment (MauPalantir) [reply post-install.xql:83]: oh right, you can just use MERMEID_exist_endpoint, so essentially this is more of a documentation issue.

**Comment by peterstadler** [comment]
This PR fixes the initial collection permissions, which is great! What about merging it like so or do you plan to do more updates on this?

**Comment by MauPalantir** [comment]
@peterstadler right, I moved it back to draft because I was also I trying to implement Omar's remarks about the conditional group change (so that we only run the create-change group if a given first-run file is present). it turned out to be less trivial than I expected though, so maybe we can merge this an open a separate PR when I succeed with that.

**Pull request merged**